### ISSUE TITLE
canceling a package upgrade with the confirmation check removes the package from chocolatey

### ISSUE THREAD

**Issue by elirnm** [chocolatey/choco#780]
### What You Are Seeing?

With git and git.install installed and outdated (going from 2.8.3 to 2.8.4), I ran 'cup git' and when the installation confirmation message was shown for git.install (which ran before git) I entered '2' to cancel the update. git.install canceled, but the git package updated without prompting for confirmation itself. After the update completed, chocolatey has lost its record of git.install being a chocolatey package, although the program is still installed on my computer. 
### What is Expected?

For chocolatey to remember packages despite canceled updates.
### How Did You Get This To Happen? (Steps to Reproduce)

install git and git.install
run 'cup git'
when prompted for confirmation, enter 2 to cancel
notice that chocolatey thinks git.install is not installed
### Output Log

https://gist.github.com/elirnm/2730b14f2900f793bfa8a83028950dcc
### Screenshot

![capture](https://cloud.githubusercontent.com/assets/19817004/15892738/3beb0584-2d30-11e6-9fce-059dcd6bf350.PNG)

**Comment by bartgenuit** [comment]
I had something similar while upgrading libreoffice. I forgot the '-y' switch, and halfway I decided I was tired of confirming all packages manually, so I bailed out (CTRL-C) to retry with the '-y'. However, by bailing out, chocolatey forgot about libreoffice completely.

Steps to reproduce:
```
> choco upgrade all
bail out by pressing CTRL-C at the confirmation dialog (in order to restart using -y switch)
> choco upgrade libreoffice
libreoffice is not installed. Installing...
```

My log: https://gist.github.com/bartgenuit/a9b3b192820c42f3bfafd7747187f772
